### PR TITLE
Doc improvements

### PR DESCRIPTION
**Pull request by rechvs**
- [x] You have read the [Spring Data contribution guidelines](https://github.com/spring-projects/spring-data-build/blob/master/CONTRIBUTING.adoc).
- [ ] (not applicable) You use the code formatters provided [here](https://github.com/spring-projects/spring-data-build/tree/master/etc/ide) and have them applied to your changes. Don’t submit any formatting related changes.
- [ ] (not applicable) You submit test cases (unit or integration tests) that back your changes.
- [ ] (not applicable) You added yourself as author in the headers of the classes you touched. Amend the date range in the Apache license header if needed. For new types, add the license header (copy from another file and set the current year only).

I kept the changes in separate commits mainly to improve reviewability. I’d be happy to squash them before merge.

### CODE REVIEW

**Comment by rechvs** [comment]
@pivotal-cla This is an Obvious Fix

**Comment by mp911de** [comment]
Care to squash your commits into a single one for easier merging and reviewing?

**Comment by rechvs** [comment]
> Care to squash your commits into a single one for easier merging and reviewing?

@mp911de  Done.

File: src/main/antora/modules/ROOT/pages/mongodb/mapping/mapping.adoc [hunk 1]
```diff
@@ -54,23 +54,23 @@ The following outlines what field will be mapped to the `_id` document field:
 | `@Id` `String` x
 | `_id`
 
-| `@Field("x")` `@Id` `String` x
-| `_id`
+| `@Field("x")` `@Id` `String` y
+| `x`
```
Comment (mp911de) [inline src/main/antora/modules/ROOT/pages/mongodb/mapping/mapping.adoc:58]: This is incorrect, `@Id` takes precedence. I will update the change during the merge.

**Pull request closed without merging**